app/src/main/java/com/arygm/quickfix/model/profile/Profile.kt

```diff
@@ -37,12 +37,14 @@ class UserProfile(
 }
 
 class WorkerProfile(
+    val rating: Double = 0.0,
+    val reviews: List<String> = emptyList(),
     val fieldOfWork: String = "",
     val hourlyRate: Double? = null,
     val description: String = "",
-    val location: Location?,
+    val location: Location? = null,
```

Comment:
For the document to worker it's fine i've tested it
For the document to user, you're saying that 
`val locationsData = document.get("locations") as? List<Map<String, Any>> ?: emptyList()` instead of 
`val locationsData = document.get("locations") as? List<Map<String, Any>> ?: return null` should be fine ?